src/utils/log.rs

```diff
@@ -58,3 +61,56 @@ unsafe extern "C" fn log_callback(importance: wlr_log_importance,
     });
     RUST_LOGGING_FN(importance, message);
 }
+
+pub struct Logger;
+
+impl Logger {
+    pub fn init(level: LevelFilter) {
```

Comment:
Is there a reason we can't expose passing an `Into<Option<LogCallback>>` like in `init_logging`? This would easily allow the compositor writer to still do something special whenever there is a log from the wlroots logging.